.github/workflows/build-sample-apps.yml

```diff
@@ -0,0 +1,126 @@
+name: Build sample apps 
```

Comment:
This file was taken from the iOS SDK and modified to build Android apps. 

There is a possibility that we can re-use the contents in this file with all of the SDK wrapper projects but I am not sure yet. That is my next task to determine with this ticket.